grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -157,8 +157,8 @@ class StockMovementApiController {
     def reviseItems = {
         StockMovement stockMovement = stockMovementService.getStockMovement(params.id)
         bindStockMovement(stockMovement, request.JSON)
-        def revisedItems = stockMovementService.reviseItems(stockMovement)
-        render([data: revisedItems] as JSON)
+        stockMovementService.reviseItems(stockMovement)
+        render([data: 200] as JSON)
```

Comment:
if we no longer want to send the revised items back via JSON, why not use `render(status: 200)` here?

Reply:
I forgot about this, thanks

Reply:
@mdpearson done